src/main/java/org/swisspush/reststorage/RestStorageHandler.java

```diff
@@ -750,6 +750,7 @@ private void respondWith(HttpServerResponse response, StatusCode statusCode, Str
         response.setStatusCode(statusCode.getStatusCode());
         response.setStatusMessage(statusCode.getStatusMessage());
         if (responseBody != null) {
+            response.putHeader(CONTENT_TYPE.getName(), "text/plain");
```

Comment:
It is always text/plain here?

Reply:
Yes when provided it's always a String error message